src/main/java/study/race/domain/CarName.java

```diff
@@ -0,0 +1,27 @@
+package study.race.domain;
+
+public class CarName {
+
+    private int maxCarNameLength = 5;
```

Comment:
상수는 다음과 같이 해주면 어떨까요??
```suggestion
    private static final int LENGTH = 5;
```